src/main/java/org/broadinstitute/hellbender/tools/walkers/ReferenceConfidenceVariantContextMerger.java

```diff
@@ -28,12 +30,17 @@
 public final class ReferenceConfidenceVariantContextMerger {
 
     private final GenotypeLikelihoodCalculators calculators;
+    private final VCFHeader vcfInputHeader;
     protected final VariantAnnotatorEngine annotatorEngine;
     protected final OneShotLogger warning = new OneShotLogger(this.getClass());
+    protected final OneShotLogger oneShotHeaderLineLogger = new OneShotLogger(this.getClass());
```

Comment:
While you're at it, could you rename warning to something like `nonNumericalAttributeWarning` or something to that end so its clearer in the future.